javascript-algorithms-and-data-structures/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/styles.css

```
@@ -113,7 +107,7 @@ td {
   background-color: var(--light-orange);
 }
 
-.general {
+.General {
```

This seems to be the only class selector that starts with a capital letter.

To keep this consistent, it can be lowercased:

```suggestion
.general {
```